pkg/runner/flags.go

```diff
@@ -64,7 +64,7 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 	fs.StringVar(&cfg.ThanosStoreAPIListenAddr, "thanos-store-api-listen-address", "", "Address to listen on for Thanos Store API endpoints.")
 	fs.StringVar(&corsOriginFlag, "web-cors-origin", ".*", `Regex for CORS origin. It is fully anchored. Example: 'https?://(domain1|domain2)\.com'`)
 	fs.Int64Var(&cfg.HaGroupLockID, "leader-election-pg-advisory-lock-id", 0, "(DEPRECATED) Leader-election based high-availability. It is based on PostgreSQL advisory lock and requires a unique advisory lock ID per high-availability group. Only a single connector in each high-availability group will write data at one time. A value of 0 disables leader election.")
-	fs.DurationVar(&cfg.ThroughputInterval, "tput-report", time.Second, "Interval in seconds at which throughput should be reported. Setting it to '0' will disable reporting throughput.")
+	fs.DurationVar(&cfg.ThroughputInterval, "tput-report", time.Second, "Setting duration to `0` will disable reporting throughput, else interval with unit at which throughput should be reported, e.g. `10s` or `3m.`")
```

Comment:
Let's reflect the same in the actual code too.

```suggestion
	fs.DurationVar(&cfg.ThroughputInterval, "tput-report", time.Second, "Duration interval at which throughput should be reported. Setting duration to `0` will disable reporting throughput, otherwise, an interval with unit must be provided, e.g. 10s or 3m.")
```